uint8 left_motor
uint8 right_motor